recipes/journey_1/module_3/probe.yaml

```diff
@@ -132,14 +159,23 @@ tasks:
         edges:
           - text: "Give me an example, Ada!"
             target_node_id: END
+          - text: "On to the next, Ada!"
+            target_node_id: screen3b
+      - id: screen3b
+        type: message
+        body:
+          parts:
+            - type: "text"
+              content: "For instance, instead of asking 'Write code for a website,' which is too vague and open-ended, a better prompt would be 'Write HTML, CSS, and JavaScript code for a responsive single-page website with a navigation bar, image gallery, and contact form.' This detailed prompt gives the AI clear instructions on what is required, leading to more relevant and useful code generation."
+        edges:
           - text: "On to the next, Ada!"
             target_node_id: END
       - id: END
```

Comment:
Please see the CTA confusion here

![image](https://github.com/user-attachments/assets/0a01e095-1c97-4be8-83fc-e62005ed9948)